agents/coder.js

```diff
@@ -20,8 +49,6 @@ YOUR TASK: As a senior software developer, make the requested changes from the U
 RESPONSE FORMAT: This is the format of your reply. 
 Provide a new version of the source code with the task complete.
 Code only. No comments or other text. 
```

Comment:
From my tests, this is making gpt to delete the comments that already exist in the code

Reply:
that's gpt4 vs gpt3.5 it seems
we agree to focus on gpt3.5 for now

the actual goal is to get local models in the future
and we should offer some prompt flexibility including alternatives by a model I guess for really far futures, called `Prompt Selectors` in langchain 
https://js.langchain.com/docs/modules/chains/prompt_selectors/